docs/cli/nrfu.md

```diff
@@ -145,6 +145,29 @@ anta nrfu --tags LEAF json
 ```
 ![$1anta nrfu json results](../imgs/anta-nrfu-json-output.png){ loading=lazy width="1600" }
 
+## Performing NRFU and saving results in a CSV file.
+
+The `csv` command in NRFU testing is useful for generating a CSV file with all tests result. This file can be easily analyzed and filtered by operator for reporting purposes.
+
+### Command overview
+
+```bash
+anta nrfu csv --help
+Usage: anta nrfu csv [OPTIONS]
+
+  ANTA command to check network states with CSV result.
+
+Options:
+  --csv-output FILE  Path to save report as a CSV file. It only saves test
+                     results and not the output from --group-by option  [env
+                     var: ANTA_NRFU_CSV_CSV_OUTPUT]
```

Comment:
I would move `It only saves test
                     results and not the output from --group-by option` to the command help rather than here

Reply:
Agreed, but it will be only visible with `anta nrfu csv --help`

```bash
❯ anta nrfu --help G csv
  csv         ANTA command to check network states with CSV result.
❯ anta nrfu csv --help
Usage: anta nrfu csv [OPTIONS]

  ANTA command to check network states with CSV result. It only saves test
  results and not the output from --group-by option.

Options:
  --csv-output FILE  Path to save report as a CSV file  [env var:
                     ANTA_NRFU_CSV_CSV_OUTPUT]
  --help             Show this message and exit.
```

Reply:
yes but group-by is anyway only available using `anta nrfu table --help` so now I am even questioning adding this